crates/nargo_cli/src/cli/codegen_verifier_cmd.rs

```diff
@@ -35,8 +36,8 @@ pub(crate) fn run(args: CodegenVerifierCommand, config: NargoConfig) -> Result<(
         }
     };
 
-    #[allow(deprecated)]
-    let smart_contract_string = backend.eth_contract_from_cs(preprocessed_program.bytecode);
+    let smart_contract_string =
+        backend.eth_contract_from_vk(&preprocessed_program.verification_key);
```

Comment:
Can you switch this to use the `codegen_verifier` function from `nargo` lib?

Reply:
Done